### DISCUSSION TITLE
enlazar mi ejercicio resuelto con git

### DISCUSSION THREAD

**Discussion by AaronMiranda2021** [atralice/Curso.Prep.Henry#57] [Q&A]
Hola profe, tengo un problema me aparece lo siguiente cuando trato de enlazar los ejercicios resueltos del homework JS-01

aaron@LAPTOP-DBJIPL33 MINGW64 ~/OneDrive/Escritorio/Demo/Curso.Prep.Henry/02-JS-I/homework (master)
$ git push origin master
info: please complete authentication in your browser...
remote: Permission to atralice/Curso.Prep.Henry.git denied to AaronMiranda2021.
fatal: unable to access 'https://github.com/atralice/Curso.Prep.Henry.git/': The requested URL returned error: 403

como que se conectó con su repo "atralice/Curso.Prep.Henry" y me denegaron en mi repo,  "AaronMiranda2021/Curso.Prep.Henry"

**Comment by jjbagus** [comment]
Te recomiendo que ingreses al Slack (te deberían haber enviado una invitación) y ahí te van a poder ayudar más rápido y mejor.

**Reply by AaronMiranda2021** [reply]
una invitación de que?? buen gracias!

**Reply by jjbagus** [reply]
Te cuento, para que no te pase lo mismo que a mí, que hice todo el curso pre challenge sin saber que existía el Slack!
Slack es una aplicación en la que todo el alumnado comparte espacio con los profesores, van compartiendo dudas y resolviendo en comunidad. Además por lo que pude ver (recién me metí hoy por primera vez) también hay zoom de los profes resolviendo ejercicios y demás. No te lo pierdas! yo pensaba que slack era este espacio, pero nadie le da bola a esto, y ayer un compañero me respondió. ( si ves mi pregunta (donde están todos?)
Chequea en la carpeta de SPAM si no te llegó la invitación a SLACK. Si no te llegó, escribiles a admisiones@soyhenry.com que te contestan al toque y te invitan.
Mucha suerte con eso!
